### DISCUSSION TITLE
How to set the background color of the LSP hover information "window"?

### DISCUSSION THREAD

**Discussion by mlauronen** [LunarVim/LunarVim#4405] [Q&A] [answered]
The current background color is the same as the default background color, so it's a bit hard to read.
Please see the example: [https://imgur.com/gZBLhsB.png](https://imgur.com/gZBLhsB.png)

**EDIT**
tl;dr;
Change the colorscheme to one which works ;) Thanks.

**Comment by BtAbdul** [comment]
I think its default and follows the color scheme color,
Maybe you can change it by cmp.nvim  plugin

**Comment by mlauronen** [comment]
I'm using an unmodified version of LunarVim (fresh install), so colorchme defaults to "lunar". It seems that the problem is with all windows that don't have borders.

I'm not sure what is the right way to fix this without fixing all the plugins one by one. So if any dev who sees this could give me some directions how to start to fix this would be a great help.

**Reply by BtAbdul** [reply, marked as answer]
![IMG_20231106_195829](https://github.com/LunarVim/LunarVim/assets/46757797/4088b28d-0c93-4029-b754-1b7f37341ac1)

you can try these themes,
[Catppccin : because its best 😇](https://github.com/catppuccin/nvim)

[Nightfly : its best](https://github.com/bluz71/vim-nightfly-colors)

**Reply by mlauronen** [reply]
Thanks for the idea.

The background color was too light so I had to make a small change for it in the **config.lua**

```
lvim.plugins = {
  {
    "catppuccin/nvim",
    config = function()
      require("catppuccin").setup {
          color_overrides = {
              mocha = {
                  base = "#151515"
              }
          }
      }
      lvim.colorscheme = "catppuccin-mocha"
    end
  }
}
```

**Reply by BtAbdul** [reply]
That's good.

catppuccin offers these themes,
- Latte
- Frappe
- Macchiato
- Mocha

to set theme add follow in config.lua
`lvim.colorscheme = "catppuccin"` 
or
`lvim.colorscheme = "catppuccin-frappe"`

or you can change theme in lvim for temporary, type command
`:colorscheme catppuccin-mocha`

**Reply by mlauronen** [reply]
Alright. I think the plugin's spec is a good place to do that, as you can see in my code example above.
